samples/meeting-live-coding-interview/csharp/README.md

```diff
@@ -0,0 +1,93 @@
+---
+page_type: sample
+description: This sample demos live coding in a teams meeting."
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+  contentType: samples
+  createdDate: "24/3/2022 2:36:57 PM"
+---
+
+# Live coding interview using Shared meeting stage 
+
+This sample demos live coding in a teams meeting. In side panel there is a list of question in specific coding language and on share click specific question with language code editor will be shared with other participant in meeting.
+Now any participant in meeting can write code for the question and same will be updated to all the other participants in meeting.
+
+![side panel ](MeetingLiveCoding/Images/sidePanelView.png)
+
+![shared content](MeetingLiveCoding/Images/stageView.png)
+
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  ```bash
+  # run ngrok locally
+  ngrok http -host-header=localhost 3978
+  ```
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+
+2) In a terminal, navigate to `samples/meeting-live-coding-interview/csharp`
+
+    ```bash
+    # change into project folder
+    cd # MeetingLiveCoding
+    ```
+3) Run ngrok - point to port 3978
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+4) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details
+   - `<<App-ID>>` with some unique GUID   
+   - `<<BASE-URL>>` with your application's base url, e.g. https://1234.ngrok.io
+   - `<<VALID DOMAIN>>` with your app domain e.g. *.ngrok.io
+
+5) Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams.
+
+6) Run the app from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
+
+  ```bash
+  # run the app
```

Comment:
Have you tried this with a clean clone? I think there might be a need to call out the requirement for `npm install` somewhere